.github/ISSUE_TEMPLATE/register-rococo-parachain.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Register Rococo Parachain
+about: Request parachain registration from the adminstrators of the Rococo test network
+title: "[PARACHAIN NAME] Rococo Registration"
+labels: rococo-registration
+assignees: ''
+
+---
+
+# About Your Parachain
+
+- Name: 
+- ID: 
+- Purpose: 
+- GitHub: 
```

Comment:
I think we need a riot handle as well IMHO.

Reply:
See below :wink: